### PR TITLE
PD-679: Configure access to the PX-Backup UI on OpenShift

### PR DESCRIPTION
**Pull request by andreipope**

### CODE REVIEW

File: content/use-px-backup/configure-ui.md [hunk 1]
```diff
@@ -132,3 +133,25 @@ You can access the Keycloak UI at the `/auth` path:
...
+### Access the PX-Backup UI using a route
+
+1. Open the web console, go to  **Networking** > **Routes**, and then select the **Create Route** button.
+
+2. On the **Create Route** page, configure your route by populating the following fields:
+    - **Name**: enter a descriptive name
+    - **Hostname**: specify a public hostname. If you leave this field empty, OpenShift will generate a hostname.
+    - **Path**: leave this field unchanged.
+    - **Service**: choose `px-backup-ui` from the drop-down list.
+    - **Target Port**: choose `80 -> 80` <!-- Not sure. Needs validation as PX-Backup is not deployed onto https://console-openshift-console.vinayak-vpc-ocp4-3-80e1e1cd66534115bf44691bf8f01a6b-0000.us-south.containers.appdomain.cloud/dashboards -->
```
Comment (andreipope) [inline content/use-px-backup/configure-ui.md:153]: @bhavana-ar @tusharraut1994

File: content/use-px-backup/configure-ui.md [hunk 2]
```diff
@@ -132,3 +133,25 @@ You can access the Keycloak UI at the `/auth` path:
...
+
+3. When you've finished configuring your route, select the **Create button**.
+
+<!-- Are there any other steps? I can't verify this as PX-Backup is not deployed onto https://console-openshift-console.vinayak-vpc-ocp4-3-80e1e1cd66534115bf44691bf8f01a6b-0000.us-south.containers.appdomain.cloud/dashboards -->
```
Comment (andreipope) [inline content/use-px-backup/configure-ui.md:157]: @bhavana-ar @tusharraut1994

File: content/use-px-backup/configure-ui.md (hunk 1 above)
Comment (bhavana-ar) [reply content/use-px-backup/configure-ui.md:153]: choose 80 -> 8080

File: content/use-px-backup/configure-ui.md (hunk 2 above)
Comment (bhavana-ar) [reply content/use-px-backup/configure-ui.md:157]: That would be the last step to setup the route. To access PX-backup they need to click on the location link in the route.

**Review by bhavana-ar — APPROVED** [review]

**Pull request merged**